supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -51,23 +51,61 @@ default   active   yes         yes
 For commands and command outputs in code blocks, observe the correct markup for user-replaced values, as described in xref:user-replaced-values[] and xref:user-replaced-values-xml[].
 
 [[explain-commands-variables-in-code-blocks]]
-== Explain commands and variables in code blocks
+=== Explanation of commands and variables used in code blocks
 
-To explain commands, lines of code, or variables (user-replaced values) used in a code block, follow the code block with the relevant explanation or description of the elements.
+To explain commands, lines of code, or variables (user-replaced values) in a code block, do not use callouts. DITA does not support callouts, hence there is no semantic markup for an AI agent to interpret or output them correctly.
 
-. Use a simple sentence to explain or describe a single line of command, variable, option, or parameter.
+Instead, follow the code block with the relevant explanation or description of the elements, using the following guidelines.
+
+* Use a simple sentence to explain or describe a single line of command, variable, option, or parameter.
 +
 .Example AsciiDoc: A simple sentence explaining a single command
-+
+
 [source,terminal]
 ----
 $ hcp create cluster <platform> --help 
 ----
 +
 Use the `hcp create cluster` command to create and manage hosted clusters. The supported platforms are `aws`, `agent`, and `kubevirt`.
+
+* Use a definition list to explain multiple options, parameters, variables, placeholders, or UI elements.
+**  List the parameters/variables in the order in which they appear in the code block.
+**  Introduce definition lists with "where:" and begin each variable description with "Specifies".
 +
-. Use a bulleted list to describe the structure of a sample YAML file or explain multiple lines of code in a code block.
+.Example AsciiDoc: A definition list explaining user-replaced values
+
+[source,yaml,subs="+attributes,+quotes"]
+----
+$ cat <<EOF | oc -n <my_product_namespace> create -f -
+apiVersion: v1
+kind: Secret
+metadata:
+ name: <my_product_database_certificates_secrets> 
+type: Opaque
+stringData:
+ postgres-ca.pem: |-
+  -----BEGIN CERTIFICATE-----
+  <ca_certificate_key> 
+ postgres-key.key: |-
+  -----BEGIN CERTIFICATE-----
+  <tls_private_key> 
+ postgres-crt.pem: |-
+  -----BEGIN CERTIFICATE-----
+  <tls_certificate_key> 
+  # ...
+EOF
+----
++
+where:
```

Comment:
This needs to be a full sentence to comply with general guidance, accessibility, and localization requirements.

```suggestion
Use the following values in the configuration file:
```

Reply:
@jafiala Because IBM Style recommends "where", I think we should keep this guidance as is. I understand your concerns about accessibility and localization. Please open a new [issue](https://github.com/redhat-documentation/supplementary-style-guide/issues), and we can discuss the pros and cons there. 

@Preeticp It's my understanding that these examples are guidance and that we can choose to deal with the description list differently. Is that correct? Would it be okay if Jan's team decides to use a complete sentence instead of "where"? Like this:

`Use the following values in the command:`